WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -492,14 +488,14 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
```

Comment:
I found the design to be similar (icons & text in rows) . Also we can always override specific styles if we want for this app.